website/docs/r/search_analyzer.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: search analyzer"
+sidebar_current: "docs-mongodbatlas-resource-search-analyzer"
+description: |-
+Provides a Search Analyzers resource.
+---
+
+# mongodbatlas_search_analyzer
+
+`mongodbatlas_search_analyzer` creates new user-defined Atlas Search [analyzers](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/#std-label-analyzers-ref) and replaces existing ones. If you have any user-defined analyzers for your Atlas Search [index](https://docs.atlas.mongodb.com/atlas-search/#std-label-fts-top-ref), they will be replaced by the new analyzer or analyzers specified in this request.
+
+## Example Usage
+
+```hcl
+resource "mongodbatlas_search_analyzer" "test" {
+  project_id = "<PROJECT_ID>"
+  cluster_name = "<CLUSTER_NAME>"
+  
+  search_analyzers {
+    base_analyzer = "lucene.standard"
+    ignore_case = yes
+    max_token_length = 10
+    name = "search_analyzer_1"
```

Comment:
This should definitely include an example for stem_exclusion_set and stopwords as those are often quite large and the format will be important.  Normally we want a bit more complete example than just the basic.